src/views/SignTransactionLedger.vue

```diff
@@ -17,18 +20,19 @@
                     :address="recipientDetails.address"
                     :label="recipientDetails.label"
                     :image="recipientDetails.image"
-                    @click.native="shownAccountDetails = recipientDetails"
+                    :displayAsCashlink="recipientDetails.isCashlink"
+                    @click.native="shownAccountDetails = !recipientDetails.isCashlink ? recipientDetails : null"
```

Comment:
I overlooked the negating `!` at first and was confused, so I suggest to turn this condition around:
```suggestion
                    @click.native="shownAccountDetails = recipientDetails.isCashlink ? null : recipientDetails"
```